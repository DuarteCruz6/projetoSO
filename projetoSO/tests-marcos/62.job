WRITE [(n3,81)]
DELETE [n3]
BACKUP
WRITE [(n3,2)]
WRITE [(d9,49)(l1,43)(v7,68)(c0,88)]
WRITE [(z5,13)(e0,52)(o3,20)]
SHOW
WRITE [(h3,80)]
DELETE [z5,h3,c0]
WRITE [(q1,29)(s1,76)(s7,58)]
WAIT 1000
DELETE [l1]
WRITE [(u6,10)(o1,0)(c1,86)]
READ [o1]
READ [v7]
READ [u6]
WRITE [(z9,56)(l0,31)]
WRITE [(j5,54)(x5,25)]
DELETE [n3,x5]
WRITE [(l2,63)(d3,46)(u0,26)(g4,77)]
DELETE [o1,s1,d9,z9,s7]
WRITE [(y9,36)(v1,43)(a0,1)(w1,99)]
READ [d3]
WRITE [(r3,42)(h0,85)(p9,10)(g0,39)]
DELETE [h0,u6,g0,g4,r3]
READ [o3]
WRITE [(e4,80)(t8,16)(w3,94)(s8,61)]
DELETE [p9]
BACKUP
WRITE [(c7,65)(k7,4)(s1,84)(i5,74)]
WRITE [(m2,71)(n7,17)]
SHOW
WRITE [(b7,27)(r1,32)(m4,41)(u7,4)]
READ [k7]
WRITE [(r8,51)(h3,87)]
DELETE [e0,b7,q1,v1,w3]
WRITE [(x5,11)(a4,49)(o9,24)]
WRITE [(a9,59)]
READ [v7]
WRITE [(q1,35)(k9,73)(r9,37)]
DELETE [m2,e4]
WRITE [(v4,44)(t1,10)]
READ [w1]
WRITE [(k6,78)]
DELETE [x5,d3,k7,s1,y9]
WRITE [(u6,15)]
READ [t1]
READ [m4]
DELETE [a4,r8]
WRITE [(f6,21)(m2,49)(s4,51)(l4,31)]
DELETE [m4,l2]
DELETE [v4]
WAIT 3000
READ [m2]
DELETE [s4,u0,n7]
SHOW
WRITE [(n6,5)(k5,12)]
WRITE [(l2,24)(a5,45)]
WRITE [(j1,80)(d3,71)(h2,50)(e1,80)]
WRITE [(m9,97)(z5,66)(f0,43)(s0,22)]
WRITE [(t4,86)(z7,24)(v4,9)(u8,1)(l5,34)]
READ [a9]
READ [h2]
READ [z5]
WRITE [(a1,10)(a8,50)(z0,4)]
WRITE [(p4,31)(c0,44)(w7,72)]
WRITE [(d0,75)]
WRITE [(s2,93)]
WRITE [(j9,68)(u9,49)]
WRITE [(x5,44)(v0,9)(m3,27)]
WRITE [(r0,64)(m7,67)(g0,23)]
WRITE [(z2,44)(x6,67)(q6,1)(s4,5)(x3,65)]
WRITE [(u2,0)]
WRITE [(c6,53)(y7,49)]
DELETE [n6,r1,z0,j9]
READ [s2]
DELETE [u2,x6,c1,t8]
WRITE [(n0,82)(b2,15)(y4,100)]
READ [a0]
WRITE [(k4,42)(r8,30)(d1,30)]
WRITE [(j6,94)(b9,97)(m6,61)(r2,30)]
WRITE [(n9,72)(l3,82)(l6,46)(z3,48)(t5,66)]
DELETE [o3,a5,t5]
WRITE [(u3,9)(f3,91)]
WRITE [(v8,68)(o6,80)(c4,6)(i1,23)(i6,52)]
DELETE [n0,e1,g0]
WRITE [(g7,28)]
WRITE [(q8,1)]
WRITE [(d5,45)(y9,59)(f8,39)]
WRITE [(c3,60)(a2,57)(s9,98)]
READ [d1]
BACKUP
READ [l0]
WRITE [(o2,74)(u2,33)]
WRITE [(p7,97)(i8,38)]
WRITE [(t0,4)]
BACKUP
WRITE [(x2,52)]
READ [l3]
WRITE [(u4,54)(d8,78)(h1,73)(o4,19)(b1,35)]
