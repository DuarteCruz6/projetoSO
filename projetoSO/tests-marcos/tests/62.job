WAIT 3000
WRITE [(e9,85)(u0,65)(z9,94)(h0,0)]
WRITE [(r9,34)(r3,45)(q0,86)(y1,23)]
WRITE [(p4,24)(g1,51)]
DELETE [u0,z9,e9]
WRITE [(x9,41)(g7,83)(u4,68)(e2,26)(v8,62)]
DELETE [x9,p4,r9,q0]
DELETE [y1,v8,g1,r3,e2]
READ [h0]
WRITE [(x9,42)]
WRITE [(a8,26)(x2,86)(e3,41)(e4,58)]
BACKUP
WRITE [(c8,48)(f3,55)(t6,59)]
WRITE [(i9,7)(s2,58)(c9,64)(r0,10)]
READ [i9]
WRITE [(m2,48)(u1,26)(l7,47)(y6,92)(i7,51)]
WRITE [(l6,81)(r7,38)(w9,39)(c0,9)]
READ [y6]
READ [w9]
WRITE [(r4,79)]
WRITE [(i6,35)(o7,26)(d1,16)(m8,80)]
DELETE [u4,a8,i6]
WRITE [(q7,14)(q7,31)(o1,29)]
READ [g7]
WRITE [(n7,59)(s1,53)(a5,94)(r3,58)]
READ [y6]
READ [l7]
WRITE [(u3,93)(r2,50)(h5,22)(p8,89)]
BACKUP
WAIT 1000
WRITE [(e8,82)(o2,67)(s9,24)(a8,17)]
READ [n7]
WRITE [(o6,49)(g8,96)]
WRITE [(i5,56)(l4,50)]
READ [r7]
WRITE [(c4,48)]
DELETE [c9,m2]
WRITE [(u2,38)]
WRITE [(u6,72)(z4,60)(a1,91)(w3,87)(w4,86)]
SHOW
WRITE [(n1,37)(v7,72)(z9,93)(u8,1)(l5,54)]
READ [p8]
READ [c8]
DELETE [r4,c8,p8,f3,o1]
BACKUP
WRITE [(s0,53)(n5,87)(p0,5)(k9,24)(g1,62)]
READ [a1]
DELETE [t6,u1,q7,s9,n5]
WRITE [(p8,32)]
WAIT 3000
SHOW
WRITE [(x7,5)(o1,17)]
WRITE [(j8,41)(h2,72)(q3,58)(h8,15)(h3,29)]
WRITE [(u1,32)(w1,8)(n4,40)(y0,14)]
SHOW
WRITE [(e2,68)(q8,83)(c2,35)(a6,18)]
READ [k9]
READ [n1]
WRITE [(p3,98)]
WRITE [(e9,39)(s8,62)(q7,24)(c5,24)(e9,20)]
SHOW
WRITE [(j5,99)(v8,23)(z6,34)(m3,71)]
WRITE [(d3,10)(y8,0)(b5,5)(r6,97)(h6,87)]
WRITE [(k3,93)(l8,83)(t8,11)]
DELETE [q8]
