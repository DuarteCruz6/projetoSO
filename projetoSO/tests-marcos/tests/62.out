[(h0,0)]
[(i9,7)]
[(y6,92)]
[(w9,39)]
[(g7,83)]
[(y6,92)]
[(l7,47)]
[(n7,59)]
[(r7,38)]
(a1, 91)
(a8, 17)
(a5, 94)
(c4, 48)
(c0, 9)
(c8, 48)
(d1, 16)
(e8, 82)
(e4, 58)
(e3, 41)
(f3, 55)
(g8, 96)
(g7, 83)
(h5, 22)
(h0, 0)
(i5, 56)
(i7, 51)
(i9, 7)
(l4, 50)
(l6, 81)
(l7, 47)
(m8, 80)
(n7, 59)
(o6, 49)
(o2, 67)
(o1, 29)
(o7, 26)
(p8, 89)
(q7, 31)
(r2, 50)
(r3, 58)
(r4, 79)
(r7, 38)
(r0, 10)
(s9, 24)
(s1, 53)
(s2, 58)
(t6, 59)
(u6, 72)
(u2, 38)
(u3, 93)
(u1, 26)
(w4, 86)
(w3, 87)
(w9, 39)
(x2, 86)
(x9, 42)
(y6, 92)
(z4, 60)
[(p8,89)]
[(c8,48)]
[(a1,91)]
(a1, 91)
(a8, 17)
(a5, 94)
(c4, 48)
(c0, 9)
(d1, 16)
(e8, 82)
(e4, 58)
(e3, 41)
(g1, 62)
(g8, 96)
(g7, 83)
(h5, 22)
(h0, 0)
(i5, 56)
(i7, 51)
(i9, 7)
(k9, 24)
(l5, 54)
(l4, 50)
(l6, 81)
(l7, 47)
(m8, 80)
(n1, 37)
(n7, 59)
(o6, 49)
(o2, 67)
(o7, 26)
(p8, 32)
(p0, 5)
(r2, 50)
(r3, 58)
(r7, 38)
(r0, 10)
(s0, 53)
(s1, 53)
(s2, 58)
(u8, 1)
(u6, 72)
(u2, 38)
(u3, 93)
(v7, 72)
(w4, 86)
(w3, 87)
(w9, 39)
(x2, 86)
(x9, 42)
(y6, 92)
(z9, 93)
(z4, 60)
(a1, 91)
(a8, 17)
(a5, 94)
(c4, 48)
(c0, 9)
(d1, 16)
(e8, 82)
(e4, 58)
(e3, 41)
(g1, 62)
(g8, 96)
(g7, 83)
(h3, 29)
(h8, 15)
(h2, 72)
(h5, 22)
(h0, 0)
(i5, 56)
(i7, 51)
(i9, 7)
(j8, 41)
(k9, 24)
(l5, 54)
(l4, 50)
(l6, 81)
(l7, 47)
(m8, 80)
(n4, 40)
(n1, 37)
(n7, 59)
(o1, 17)
(o6, 49)
(o2, 67)
(o7, 26)
(p8, 32)
(p0, 5)
(q3, 58)
(r2, 50)
(r3, 58)
(r7, 38)
(r0, 10)
(s0, 53)
(s1, 53)
(s2, 58)
(u1, 32)
(u8, 1)
(u6, 72)
(u2, 38)
(u3, 93)
(v7, 72)
(w1, 8)
(w4, 86)
(w3, 87)
(w9, 39)
(x7, 5)
(x2, 86)
(x9, 42)
(y0, 14)
(y6, 92)
(z9, 93)
(z4, 60)
[(k9,24)]
[(n1,37)]
(a6, 18)
(a1, 91)
(a8, 17)
(a5, 94)
(c5, 24)
(c2, 35)
(c4, 48)
(c0, 9)
(d1, 16)
(e9, 20)
(e2, 68)
(e8, 82)
(e4, 58)
(e3, 41)
(g1, 62)
(g8, 96)
(g7, 83)
(h3, 29)
(h8, 15)
(h2, 72)
(h5, 22)
(h0, 0)
(i5, 56)
(i7, 51)
(i9, 7)
(j8, 41)
(k9, 24)
(l5, 54)
(l4, 50)
(l6, 81)
(l7, 47)
(m8, 80)
(n4, 40)
(n1, 37)
(n7, 59)
(o1, 17)
(o6, 49)
(o2, 67)
(o7, 26)
(p3, 98)
(p8, 32)
(p0, 5)
(q7, 24)
(q8, 83)
(q3, 58)
(r2, 50)
(r3, 58)
(r7, 38)
(r0, 10)
(s8, 62)
(s0, 53)
(s1, 53)
(s2, 58)
(u1, 32)
(u8, 1)
(u6, 72)
(u2, 38)
(u3, 93)
(v7, 72)
(w1, 8)
(w4, 86)
(w3, 87)
(w9, 39)
(x7, 5)
(x2, 86)
(x9, 42)
(y0, 14)
(y6, 92)
(z9, 93)
(z4, 60)
